controllers/cloud.redhat.com/providers/kafka/strimzi.go

```diff
@@ -504,7 +504,9 @@ func (s strimziProvider) connectConfig(config *apiextensions.JSON) error {
 		"offset.storage.replication.factor":       "1",
 		"offset.storage.topic":                    "connect-cluster-offsets",
 		"status.storage.replication.factor":       "1",
-		"status.storage.topic":                    "connect-cluster-status"
+		"status.storage.topic":                    "connect-cluster-status",
+        "config.providers":                        "secrets",
+        "config.providers.secrets.class":          "io.strimzi.kafka.KubernetesSecretConfigProvider"
```

Comment:
the alignment here seems to be off. Probably tabs vs spaces?

Reply:
@wcmitchell called this out too in Slack but i think it might be browser specific which is annoying and weird. Chrome shows the lines staggered as youve called out. Firefox does not. I used spaces to line everything up as tabs did not line up perfectly in editor. Sadly go fmt is not automatically  structuring this object in either of my IDE's

![image](https://github.com/user-attachments/assets/6f4763ad-2238-451d-b7f5-5c15a7a44eac)

Reply:
I believe the current code uses tabs (I'm not loving it, but I prefer being consistent over what I prefer xD)

Reply:
Ahhh i was looking at the wrong part and thought yall meant the space between the key and value and not the line start haha. Changed the line starts to tab's to match the rest, left the spaces between key and value to match other lines as well